docs/usage/configuration/config_documentation.md

```diff
@@ -3927,6 +3927,8 @@ This setting has the following sub-options:
 * `host` and `port`: Optional host and port to use to connect to redis. Defaults to
    localhost and 6379
 * `password`: Optional password if configured on the Redis instance.
+* `dbid`: Optional redis dbid if needs to connect to specific redis logical db.
```

Comment:
Can you include a note that this was added in Synapse 1.78? See https://github.com/matrix-org/synapse/blob/1617f7881df51e8cc9fbf0d30c098a6b586138b7/docs/usage/configuration/config_documentation.md?plain=1#L445-L452 for an example.